src/run.ts

```diff
@@ -57,7 +58,7 @@ async function run() {
         namespace = 'default';
     }
     let action = core.getInput('action');
-    let manifests = manifestsInput.split('\n');
+    let manifests = manifestsInput ? manifestsInput.split('\n') : undefined;

```

Comment:
Shouldn't it be `""` or `[]` instead?

Reply:
You'll end up seeing some type mismatch down the line.

Reply:
`!manifestsInput` would never be hit due to the check in line 53. You probably should just add a `return` statement to break the action flow there.